packages/storage/src/task.ts

```diff
@@ -39,140 +38,140 @@ import {
 import { Request } from './implementation/request';
 import { UploadTaskSnapshot } from './tasksnapshot';
 import { async as fbsAsync } from './implementation/async';
-import { Location } from './implementation/location';
 import * as fbsMetadata from './implementation/metadata';
 import * as fbsRequests from './implementation/requests';
 import { Reference } from './reference';
-import { StorageService } from './service';
+import { getMappings } from './implementation/metadata';
 
 /**
  * Represents a blob being uploaded. Can be used to pause/resume/cancel the
  * upload and manage callbacks for various events.
  */
 export class UploadTask {
-  private ref_: Reference;
-  private service_: StorageService;
-  private location_: Location;
-  private blob_: FbsBlob;
-  private metadata_: Metadata | null;
-  private mappings_: fbsMetadata.Mappings;
-  private transferred_: number = 0;
-  private needToFetchStatus_: boolean = false;
-  private needToFetchMetadata_: boolean = false;
-  private observers_: Array<Observer<UploadTaskSnapshot>> = [];
-  private resumable_: boolean;
-  private state_: InternalTaskState;
-  private error_: Error | null = null;
-  private uploadUrl_: string | null = null;
-  private request_: Request<unknown> | null = null;
-  private chunkMultiplier_: number = 1;
-  private errorHandler_: (p1: FirebaseStorageError) => void;
-  private metadataErrorHandler_: (p1: FirebaseStorageError) => void;
-  private resolve_: ((p1: UploadTaskSnapshot) => void) | null = null;
-  private reject_: ((p1: Error) => void) | null = null;
-  private promise_: Promise<UploadTaskSnapshot>;
+  private _ref: Reference;
+  /**
+   * @internal
+   */
+  _blob: FbsBlob;
+  /**
+   * @internal
+   */
+  _metadata: Metadata | null;
+  private _mappings: fbsMetadata.Mappings;
+  /**
+   * @internal
+   */
+  _transferred: number = 0;
+  private _needToFetchStatus: boolean = false;
+  private _needToFetchMetadata: boolean = false;
+  private _observers: Array<StorageObserver<UploadTaskSnapshot>> = [];
+  private _resumable: boolean;
+  /**
+   * @internal
+   */
+  _state: InternalTaskState;
+  private _error: FirebaseStorageError | null = null;
+  private _uploadUrl: string | null = null;
+  private _request: Request<unknown> | null = null;
+  private _chunkMultiplier: number = 1;
+  private _errorHandler: (p1: FirebaseStorageError) => void;
+  private _metadataErrorHandler: (p1: FirebaseStorageError) => void;
+  private _resolve: ((p1: UploadTaskSnapshot) => void) | null = null;
+  private _reject: ((p1: FirebaseStorageError) => void) | null = null;
+  private _promise: Promise<UploadTaskSnapshot>;
```

Comment:
If you are bored, you may want to just use "undefined" here and the question mark token.

Reply:
Sure, why not.